### DISCUSSION TITLE
Need push notification support in China. (e.g. umeng, jiguang)

### DISCUSSION THREAD

**Discussion by sg552** [element-hq/element-meta#2637] [Ideas]
Google does not work in China, mainland. 

so currently Android does not show notification when received message.

There is only 1 option for push : google service.

Please add more support, such as; umeng, jiguang . which are chinese native push product. 

Thanks.

**Comment by SaltedFishesNG** [comment]
fcm works fine in China.
Or, you can use the UnifiedPush.
